packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -16,11 +16,19 @@ const { Icon } = StorageBrowserElements;
 
 const { Cancel, Exit, Primary, Summary, Table } = Controls;
 
-const LOCATION_ACTION_VIEW_COLUMNS: Column<CancelableTask>[] = [
+interface LocationActionViewColumns extends CancelableTask {
+  folder: string;
+}
+
+const LOCATION_ACTION_VIEW_COLUMNS: Column<LocationActionViewColumns>[] = [
```

Comment:
I think the `Column` interface I set up for the table makes adding custom columns a lot more difficult since it doesn't exist on the object type 😞

If anyone has any ideas how to set this up better, please let me know. Not sure if refactoring it now would be worth it, but am open to learning/talking about it